sql/_05_plcsql/_01_testspec/_05_bug_fix/cases/21_adddate_addtime_error_input.sql

```diff
@@ -0,0 +1,37 @@
+--+ server-message on
+-- Verifies CBRD-25273, CBRD-25303
+-- Test cases for adddate() and addtime() 
```

Comment:
plz add more info.
I don't know, why this test case is related to the CBRD-25273.

Reply:
also, it needs to be verified for CBRD-25303 case